### PR TITLE
chore(version selector): Add more spacing between under heading

### PR DESCRIPTION
**Pull request by pbortnick**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-pbupdate-version-switcher-spacing-hashicorp.vercel.app/vault/docs/secrets/kv) 🔎
- [Asana task](url) 🎟️

## 🗒️ What

Add 16px of spacing under the heading to increase space under version selector.

## 🤷 Why

Requested from design

### CODE REVIEW

**Review by thiskevinwang — APPROVED** [review]
LGTM — left a non-blocking suggestion

File: src/views/docs-view/docs-view.module.css [hunk 1]
```diff
@@ -36,5 +36,10 @@
 		& :global(.landing-hero) {
 			margin-right: 25%;
 		}
+
+		/* Add more space in between the first section after title and the version selector  */
+		& h1:first-child {
+			margin-bottom: 16px;
+		}
```
[the remark this answers is not in the document]
Comment (pbortnick) [reply src/views/docs-view/docs-view.module.css:43]: Yeah I can see that. You think there should be 16px of space regardless of if the version switcher is present or not?

[the remark this answers is not in the document]
Comment (pbortnick) [reply src/views/docs-view/docs-view.module.css:43]: I think that would make sense and create more congruity. I'll just double check with design 👍

[the remark this answers is not in the document]
Comment (thiskevinwang) [reply src/views/docs-view/docs-view.module.css:43]: Oh, I didn't immediately see the nesting within `.contentWithVersions`.
If design only wants `16px`, _when the version selector is present_, then ignore my suggestion!

**Pull request merged**